### DISCUSSION TITLE
clickhouse数据迁移到doris

### DISCUSSION THREAD

**Discussion by amy108** [apache/doris#10209] [A - General / Q&A]
用什么方式从clickhouse中迁移数据到Doris最方便

**Comment by zy-kkk** [comment]
我使用的方式里目前最方便的是用clickhouse-client的方式把查询数据导出为本地csv文件，再用Doris的Stream load将文件导入
中间需要用sed -i 's/"//g' file.csv处理下去掉导出的CSV中的双引号，不然Stream load会出错
https://clickhouse.com/docs/zh/interfaces/formats#csv
https://doris.apache.org/zh-CN/docs/data-operate/import/import-way/stream-load-manual.html
